internal/app/singularity/oci_run_linux.go

```diff
@@ -95,6 +95,9 @@ func OciRun(containerID string, args *OciArgs) error {
 	}
 
 	if err := attach(engineConfig, true); err != nil {
+		// kill container before deletion
+		sylog.Errorf("%s", err)
```

Comment:
`sylog.Errof("Unable to attach to container: %s", err)`

Reply:
Error message become a bit redundant and overly long, there is an explicit error message coming from attach